django/todo_be/todo_api/serializers.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
+from django.contrib.auth.models import User
+from .models import ToDo
+
+class UserSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = User
+        fields = ('first_name', 'last_name', 'email', 'password')
```

Comment:
mislim da je password samo read-only? Ili gresim? Da li ti je hashovao sifru kada si upisao za kreiranje? Posto nesto sumnjam da je to uradio ali proveri

Reply:
mozda i ne mora ready-only ako ga vezujes na modelviewset